explainer.md

```diff
@@ -66,26 +66,26 @@ The user agent will be aware that the format as described can’t be played at a
 
 **Would the playback be power efficient?** The user agent has a priori knowledge about the power efficiency of playback and can share this information with the web page. Power efficiency is usually associated with hardware decoding but in practice hardware decoding is just one factor alongside others like resolution. For example, at lowerresolutions, software decoded formats are usually power efficient. This is information that the application can combine with the usage of the [Battery Status API](https://w3c.github.io/battery/) [t](https://w3c.github.io/battery/)o[ ](https://w3c.github.io/battery/)m[a](https://w3c.github.io/battery/)k[e](https://w3c.github.io/battery/) [d](https://w3c.github.io/battery/)e[c](https://w3c.github.io/battery/)i[s](https://w3c.github.io/battery/)i[o](https://w3c.github.io/battery/)n[s](https://w3c.github.io/battery/).
 
-```
-navigator.mediaCapabilities.query({
+```JavaScript
+navigator.mediaCapabilities.decodingInfo({
   type: "file",
   video: {
-    type: "video/webm codec=vp9.0",
+    contentType: "video/webm; codecs=vp9.0",
```

Comment:
Thanks a ton for the fixes. One lingering issue is these vp9.0 strings. Thats an old format which we later rejected for being ambiguous. The equivalent string in the new format is vp09.00.10.08. If you use that string in place of vp9.0 here and below you will find the supported and smooth values are "true" (assuming you have decent hardware).

Reply:
@foolip, friendly ping on these nits.

Reply:
Ooops, now pushed. I *haven't* tried to run the examples again to confirm this works, so trying one before merging would be a good precaution.